app/styles/brands/psyarxiv.scss

```diff
@@ -25,5 +25,5 @@
 
 .branded-footer a,
 #view-page .osf-box-lt .fa {
-    color: #ff383f;
+    color: #CF1D35;
```

Comment:
Generally I would say yes, but we're probably going to end up pulling all provider-specific CSS into a separate CSS file that's loaded onto a CDN and addressed using a scheme rather than relying on it being pre-processable, so that change would probably end up being discarded before long.